scipy/optimize/_minimize.py

```diff
@@ -314,10 +314,10 @@ def minimize(fun, x0, args=(), method=None, jac=None, hess=None,
              Function evaluations: 64
              Gradient evaluations: 64
     >>> res.x
-    [ 1.  1.  1.  1.  1.]
-    >>> print res.message
+    array([ 1.  1.  1.  1.  1.])
+    >>> print(res.message)
     Optimization terminated successfully.
-    >>> res.hess
+    >>> res.hess_inv
```

Comment:
This looks to be just a typo: `res` does not have a `hess` attribute, it is documented to have the inverse Hessian, and `hess_inv` attribute exists  (and what I get locally agrees with the values in the docstring).